Documentation/ApiOverview/Backend/Ajax.rst

```diff
@@ -77,6 +77,43 @@ with other existing routes.
 ..  attention::
     Flushing caches is mandatory after modifying any route definition.
 
+..  _protect-ajax-endpoint:
+
+Protect the endpoint
+====================
+
+..  important::
+    AJAX routes are **not protected by default** and need proper permission
```

Comment:
That is not really correct ("not protected by default"), previously any authenticated backend user can access those AJAX routes, the part states implicitely, that also non-authenticated users may access those AJAX routes. At least, I read the commit message that way:
https://review.typo3.org/c/Packages/TYPO3.CMS/+/90624

IMHO we should clarify that.

Reply:
Thanks, you're right, it's not precise enough. What about:

```suggestion
    AJAX routes are **accessible to all authenticated backend users** by default and need proper permission
```

Reply:
Sounds good to me